services/apis/sheepdog/sheepdogProps.js

```diff
@@ -60,13 +60,16 @@ module.exports = {
   }),
 
   /**
-   * Gen3Response when no authentication provided
+   * Expected contents of the response when no authentication is provided
    */
-  resNoAuth: new Gen3Response({
-    request: {},
+  resNoAuth: {
```

Comment:
If this is a generic 401 missing auth, then maybe the message shouldn't be specifically about expiry? 
But it seems that the only place that this (and `hasNoAuthError`) are used is this one scenario in `SubmitAndQueryNodesTest`:
```
  5 Scenario('submit node unauthenticated @reqData', async (sheepdog, nodes, users) => {
  6   await sheepdog.do.addNode(nodes.getFirstNode(), users.mainAcct.expiredAccessTokenHeader);
  7   sheepdog.ask.hasNoAuthError(nodes.getFirstNode().addRes);
[..blah]
```
Since this is testing with `expiredAccessTokenHeader` and this is the only place `hasNoAuthError` is used, maybe just rename `hasNoAuthError` --> `hasExpiredAuthError` and `resNoAuth` --> `resExpiredAuth`? :pray: